README.md

```diff
@@ -19,6 +19,8 @@ Github CI
 - `kuka_lbr_iiwa_support` contains urdf, config and mesh files for KUKA LBR iiwa robots
 - `kuka_lbr_iiwa_moveit_config` contains configuration files for KUKA LBR iiwa robots necessary for planning with MoveIt.
 - `kuka_mock_hardware_interface` contains a custom mock hardware interface for KUKA robots
+- `kuka_mobile_robot_support` contains urdf, config and mesh files for KUKA Mobile robot.
```

Comment:
I think this name is too general, it should contain exactly what type of mobile robots are included